Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/5.2.0/DataStandard_520_ApprovalTests.Verify.Extensions.Sample.1.0.0_Std_5.2.0_Models_Entities_Entities.generated.approved.cs

```diff
@@ -206,23 +211,27 @@ namespace EdFi.Ods.Entities.NHibernate.BusAggregate.Sample
     /// <summary>
     /// Represents a read-only reference to the <see cref="Bus"/> entity.
     /// </summary>
+    [MessagePackObject]
```

Comment:
Reference Data is serialized so that we don't have to look up the Id/Discriminator values separately. Full support of links is coming with ODS-6557.